### PR TITLE
[slack] Improve preview UI

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Remove 'dialect' and 'created by' section
- Add 'created by' in the preview heading instead

## How was this patch tested?
Screenshot 2021-04-06 at 4 06 41 PM

### CODE REVIEW

File: desktop/core/src/desktop/lib/botserver/views.py [hunk 1]
```diff
@@ -243,7 +243,7 @@ def _make_unfurl_payload(request, url, id_type, doc, doc_type):
           "type": "section",
           "text": {
             "type": "mrkdwn",
-            "text": "\n*<{}|Open {} {} in Hue - SQL Editor>*".format(url, name, doc_type)
+            "text": "\n*<{}|Open {} {} in Hue | Created by: {}>*".format(url, name, doc_type, created_by),
```
Comment (romainr) [inline desktop/core/src/desktop/lib/botserver/views.py:246]: Could event be 

`dialect` create by ... 

maybe

Comment (romainr) [reply desktop/core/src/desktop/lib/botserver/views.py:246]: PS: please use {url} etc and not just {} as it gets not easy to read

**Review by romainr — APPROVED** [review]
Cleaner!

File: desktop/core/src/desktop/lib/botserver/views.py (hunk 1 above)
Comment (Harshg999) [reply desktop/core/src/desktop/lib/botserver/views.py]: Fixed

**Comment by Harshg999** [comment]
Dialect in link heading
Screenshot 2021-04-07 at 12 57 36 PM

File: desktop/core/src/desktop/lib/botserver/views.py [hunk 2]
```diff
@@ -243,7 +252,7 @@ def _make_unfurl_payload(request, url, id_type, doc, doc_type):
           "type": "section",
           "text": {
             "type": "mrkdwn",
-            "text": "\n*<{}|Open {} {} in Hue - SQL Editor>*".format(url, name, doc_type)
+            "text": "\n*<{url}|Open {name} {doc_type} in Hue | Dialect: {dialect} | Created by: {user}>*".format(**payload_data),
```
Comment (romainr) [inline desktop/core/src/desktop/lib/botserver/views.py:255]: | and : are not pleasant to read

`Open {name} {dialect} {doc_type} created by {user}`

?

**Review by romainr — APPROVED** [review]

File: desktop/core/src/desktop/lib/botserver/views.py (hunk 2 above)
Comment (Harshg999) [reply desktop/core/src/desktop/lib/botserver/views.py]: Fixed with a more suitable heading

**Pull request merged**